did/plugins/phabricator.py

```diff
@@ -0,0 +1,274 @@
+# coding: utf-8
+"""
+Phabricator stats for authored and reviewed differentials, aka reviews.
+
+Config example::
+
+    [phabricator]
+    type = phabricator
+    url = https://reviews.llvm.org/api/
+    token = <authentication-token>
+    token_file = <file-with-authentication-token>
+    logins = <username1>,<username2>
+
+The authentication token is *not* optional. Go to
+https://reviews.llvm.org/settings/user/<username>/page/apitokens/ and get
+yourself a "Conduit API token". The token and the actual users for which we
+query stats are decoupled, allowing you to specify more than one username.
+
+We use this endpoint for the most part
+:https://reviews.llvm.org/conduit/method/differential.revision.search/.
```

Comment:
```suggestion
https://reviews.llvm.org/conduit/method/differential.revision.search/.
```

Or was the colon here intentional?